### PR TITLE
add localizable rule

### PR DESCRIPTION
**Pull request by Stanislav2021dev**
Add localizable resources rule

Add rule which check the correct import of localizable resources
import com.wain.core.localization.R - forbidden
import com.wain.core.localization.R  as Local - forbidden
import com.wain.core.localization.R  as LocalizableResources - correct

### CODE REVIEW

File: ktlintrules/src/main/java/com/urlaunched/android/ktlintrules/LocalizableResourcesRule.kt [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    RuleAutocorrectApproveHandler {
+    override fun beforeVisitChildNodes(
+        node: ASTNode,
+        emit: (offset: Int, errorMessage: String, canBeAutoCorrected: Boolean) -> AutocorrectDecision
+    ) {
+        if (node.elementType == ElementType.IMPORT_LIST) {
+            val importStatements = node.text.split("\n").filter { it.isNotBlank() }
+
+            importStatements.forEach { import ->
+                if ((import.contains(LOCALIZATION_IMPORT) || import.contains(LOCALIZABLE_CAST)) &&
```
Comment (eddyshvetsdev) [inline ktlintrules/src/main/java/com/urlaunched/android/ktlintrules/LocalizableResourcesRule.kt]: import.contains(LOCALIZABLE_CAST) - не потрібне

File: ktlintrules/src/main/java/com/urlaunched/android/ktlintrules/LocalizableResourcesRule.kt [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        }
+    }
+
+    companion object {
+        private const val CUSTOM_RULE_ID = "ktlintrules:localizableresourses"
+        private const val LOCALIZATION_IMPORT = "core.localization.R"
+        private const val LOCALIZABLE_CAST = "as LocalizableResources"
+        private const val CORRECT_IMPORT = "core.localization.LocalizableResources"
+        private const val ERROR_MESSAGE =
+            "The import for localization resources must be import ...core.localization.LocalizableResources"
```
Comment (eddyshvetsdev) [inline ktlintrules/src/main/java/com/urlaunched/android/ktlintrules/LocalizableResourcesRule.kt]: be import -> be import as

File: ktlintrules/src/main/java/com/urlaunched/android/ktlintrules/LocalizableResourcesRule.kt (hunk 1 above)
Comment (Stanislav2021dev) [reply ktlintrules/src/main/java/com/urlaunched/android/ktlintrules/LocalizableResourcesRule.kt]: done

File: ktlintrules/src/main/java/com/urlaunched/android/ktlintrules/LocalizableResourcesRule.kt (hunk 2 above)
Comment (Stanislav2021dev) [reply ktlintrules/src/main/java/com/urlaunched/android/ktlintrules/LocalizableResourcesRule.kt]: done

**Pull request merged**